src/components/reusuables/cards/CardVerticalHover.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { Col } from "react-bootstrap"
+import codeScreens from "../../../assets/images/codeScreens.jpg"
+
+const cardOneStyle = {
+  backgroundImage: `url(${codeScreens})`,
+}
+
+const CardVerticalHover = (props) => {
```

Comment:
Destructure props like the last review.
(props) should be ({heading, subtitle, etc})